doc/classes/SplitContainer.xml

```diff
@@ -13,19 +13,24 @@
 		<method name="clamp_split_offset">
 			<return type="void" />
 			<description>
-				Clamps the [member split_offset] value to not go outside the currently possible minimal and maximum values.
+				Clamps the [member split_offsets] values to not go outside the possible minimum and maximum values, and to not overlap with each other.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="collapsed" type="bool" setter="set_collapsed" getter="is_collapsed" default="false">
-			If [code]true[/code], the area of the first [Control] will be collapsed and the dragger will be disabled.
+			If [code]true[/code], the [Control] nodes will be set to their default positions (as if all [member split_offsets] were set to [code]0[/code]) and the draggers are disabled.
 		</member>
 		<member name="dragger_visibility" type="int" setter="set_dragger_visibility" getter="get_dragger_visibility" enum="SplitContainer.DraggerVisibility" default="0">
 			Determines the dragger's visibility. See [enum DraggerVisibility] for details.
 		</member>
 		<member name="split_offset" type="int" setter="set_split_offset" getter="get_split_offset" default="0">
```

Comment:
This should be deprecated and the description should mention it's compatibility only.

Reply:
Deprecated.
Being deprecated already means it's for compatibility, so it doesn't need to be mentioned in the description, right? I don't think other deprecated properties mention it either.